tst/teststandard/permgrp.tst

```diff
@@ -105,5 +105,10 @@ gap> iso:=IsomorphismPcGroup(g);;
 gap> Length(DoubleCosets(Image(iso,g),Image(iso,s),Image(iso,s)));
 24
 
-#
+# some lattice and deductions
+gap> MinimalFaithfulPermutationDegree(PerfectGroup(IsPermGroup,7680,1));
+76
+
+# Unbind variables so we can GC memory
+gap> Unbind(g); Unbind(dc); Unbind(ac); Unbind(g); Unbind(p); Unbind(s);
```

Comment:
This is not a review, just the first thing I noticed on a quick glance. This line is not needed - see line 5 of this file. Line 5 declares all of these variables as being local to this test file, which means (I believe) that they will be unbound after the test file is run. Perhaps @ChrisJefferson (who I think added this feature) could confirm?

Reply:
Thanks -- will change.